doc/docker/Dockerfile-solr

```diff
@@ -1,4 +1,6 @@
-FROM solr:6-alpine
+FROM solr:6.6.5-alpine
```

Comment:
Bumped solr version due to security vulnerability in <v6.6.2

Reply:
yep

Reply:
Sorry, didn't know that.
Reverted.